hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestNSSummaryTaskWithFSO.java

```diff
@@ -117,6 +121,9 @@ private TestNSSummaryTaskWithFSO() {
 
   @BeforeClass
   public static void setUp() throws Exception {
+    ozoneConfiguration = new OzoneConfiguration();
+    ozoneConfiguration.setLong(OZONE_RECON_NSSUMMARY_FLUSH_TO_DB_MAX_THRESHOLD,
+        150 * 1000L);
```

Comment:
Let's set it to a small value, say 10, in the unit tests.

Reply:
done @szetszwo

Reply:
Ok set as 10.. Pls re-review @szetszwo